ingest/scripts/compare_hashes.py

```diff
@@ -127,6 +134,24 @@ def process_hashes(
     return update_manager
 
 
+def get_joint_insdc_accession(record, insdc_keys, config, take_subset=False, subset=None):
+    subset = subset or {}
+    pairs = zip(insdc_keys, config.nucleotide_sequences)
+
+    if take_subset:
+        return "/".join(
+            f"{record[key]}.{segment}" for key, segment in pairs if record.get(key) in subset
+        )
+
+    return "/".join(f"{record[key]}.{segment}" for key, segment in pairs if record.get(key))
+
+
+def get_last_joint_accession(accession: str, submitted: dict) -> JointInsdcAccession:
+    sorted_versions = sorted(submitted[accession]["versions"], key=lambda x: x["version"])
```

Comment:
```suggestion
    sorted_versions = sorted(submitted[accession]["versions"], key=lambda x: int(x["version"]))
```

just for safety in case version is not yet an int - otherwise we get issues if we ever reach version 10